ts/output/common/FontData.ts

```diff
@@ -499,6 +554,11 @@ export class FontData<C extends CharOptions, V extends VariantData<C>, D extends
    */
   protected static defaultStretchVariants: string[] = [];
 
+  /**
+   * The dynamic file data
+   */
+  protected static dynamicFiles: DynamicFileList = {};
```

Comment:
ditto

Reply:
I can do this one, but it will cause conflicts in later PRs, so I'd like to put it off until the font PRs are merged, and make the changes then, along with the other Map fixes, if that's OK.

Reply:
Ditto. Let's put it onto the *todo* stack.